Lua plugin doesn't compile

## Steps to Reproduce the Problem

Try to compile the Lua plugin.

## Expected Result

Compilation should be sucessful.

## Actual Result

```
/usr/bin/ld: ../plugins/lua/CMakeFiles/elektra-lua-objects.dir/lua.cpp.o: in function `SWIG_Lua_NewPointerObj':
~/libelektra/cmake-build-debug/src/plugins/lua/runtime.h:2470: undefined reference to `lua_newuserdatauv'
/usr/bin/ld: ../plugins/lua/CMakeFiles/elektra-lua-objects.dir/lua.cpp.o: in function `SWIG_Lua_NewPackedObj':
~/libelektra/cmake-build-debug/src/plugins/lua/runtime.h:2535: undefined reference to `lua_newuserdatauv'
```

## System Information

- Elektra Version: master
- Operating System: Manjaro Stable Version, up-to-date as of 2020-09-05
- Lua Version: 5.4.0 (specifically 5.4.0-2 from the Manjaro extra repo)

--- a/doc/news/_preparation_next_release.md
+++ b/doc/news/_preparation_next_release.md
@@ -136,7 +136,7 @@ you up to date with the multi-language support provided by Elektra.
 
 ### CMake
 
-- <<TODO>>
+- Use Lua 5.4 when available. _(Mihael Pranjić)_
 - <<TODO>>
 - <<TODO>>
 

--- a/scripts/cmake/Modules/FindLua.cmake
+++ b/scripts/cmake/Modules/FindLua.cmake
@@ -56,7 +56,7 @@ unset (_lua_library_names)
 
 # this is a function only to have all the variables inside go away automatically
 function (set_lua_version_vars)
-	set (LUA_VERSIONS5 5.3 5.2 5.1 5.0)
+	set (LUA_VERSIONS5 5.4 5.3 5.2 5.1 5.0)
 
 	if (Lua_FIND_VERSION_EXACT)
 		if (Lua_FIND_VERSION_COUNT GREATER 1)

